### DISCUSSION TITLE
Free-sort button position

### DISCUSSION THREAD

**Discussion by jaydavies16129** [jspsych/jsPsych#1922] [Q&A] [answered]
Hi there. I am having trouble trying to re-position my button in the free-sort plugin. At the moment it is showing at the bottom-centre of the screen below the sort area, but I'd like it to be at the bottom right. I have tried (unsuccessfully) to modify the plug-in, and looked into the button_html but I'm a bit stuck! 

Any help would be great :)

**Comment by becky-gilbert** [comment, marked as answer]
Hi @jaydavies16129, the free-sort 'done' button is inside of a div, and you can move the button to the right by setting that div's `text-align` CSS property to `right`. I think the easiest way to do that is to edit this line of the plugin: https://github.com/jspsych/jsPsych/blob/7b16a1d39ab38b3a01c33c9d0a2fec06524e81ee/plugins/jspsych-free-sort.js#L186

So that the `text-align: right;` CSS rule is set in the div's in-line style, like this:
```js
    html += '<div style="text-align:right;"><button id="jspsych-free-sort-done-btn" class="jspsych-btn" '+ 
```
Let us know if that works for you, and if you have any questions.

**Comment by jaydavies16129** [comment]
Hi @becky-gilbert, for some reason thats not working. I don't know if I need to alter something in the main code as well?

**Reply by becky-gilbert** [reply]
Weird! I just double-checked this using the free-sort example file from v6.3.1, and it works for me:
![image](https://user-images.githubusercontent.com/9041788/123469334-c36eb800-d5a7-11eb-992e-850b3ccdd252.png)
Maybe you have a typo somewhere? Feel free to share code and screenshots if you want more help figuring it out.

**Comment by jaydavies16129** [comment]
Hi @becky-gilbert. Apologies, this was my fault. I accidentally didn't even saved the modified plug-in... that would have helped! Problem solved, thank you so much!

**Reply by becky-gilbert** [reply]
No problem, glad we figured it out!
